x-pack/plugins/uptime/server/legacy_uptime/lib/alerts/tls_legacy.ts

```diff
@@ -10,9 +10,12 @@ import { observabilityPaths } from '@kbn/observability-plugin/common';
 import moment from 'moment';
 import { schema } from '@kbn/config-schema';
 import { ActionGroupIdsOf } from '@kbn/alerting-plugin/common';
-import { AlertInstanceContext } from '@kbn/alerting-plugin/common';
-import { Alert, GetViewInAppRelativeUrlFnOpts } from '@kbn/alerting-plugin/server';
-import { UptimeAlertTypeFactory } from './types';
+import {
+  AlertsClientError,
+  DEFAULT_AAD_CONFIG,
```

Comment:
Question: What does `AAD` stand for (Alert Action/App Data?) 😅 ?, maybe if we call it DEFAULT_ALERT_CONFIG or something similar will be easier to understand, wdyt? (maybe not the right place to change this tho)

Reply:
Alerts as data :) We can definitely consider changing to alerts in the future. Right now we still have a decent amount of code that still refers to rules as alerts (which is why we came up with a different thing to call it in the first place 🙈)

Reply:
Ah got it, I was close 😅  thanks for explaining! Yeah, sounds good and definitely not part of the PR 👍